dotcom-rendering/src/web/components/Callout/CalloutDeadline.tsx

```diff
@@ -0,0 +1,56 @@
+import { css } from '@emotion/react';
+import { palette, textSans, space } from '@guardian/source-foundations';
+import { SvgClock } from '@guardian/source-react-components';
+
+type Props = {
+	until?: number | undefined;
```

Comment:
Think you don't need the `|undefined` here, as its an optional `?:`

Reply:
Thanks - removed